### PR TITLE
fix config

### PR DESCRIPTION
**Pull request by wjhrdy**
This config was incomplete. 
The api path is needed for the server to be accessible.
Otherwise you run into problems like this.
https://old.reddit.com/r/homeassistant/comments/6rgf04/frontend_unable_to_connect/

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

**Comment by aptalca** [comment]
Hmm, sad that the linked report is from 3 years ago, yet no one notified us on any of our channels. Unfortunately we don't prowl reddit for bugs. 

Thanks so much for the PR

File: homeassistant.subdomain.conf.sample [hunk 1]
```diff
@@ -36,4 +36,14 @@ server {
         proxy_pass $upstream_proto://$upstream_app:$upstream_port;
 
     }
+    
+    location /api {
+        proxy_http_version 1.1;
```
Comment (aptalca) [inline homeassistant.subdomain.conf.sample:41]: Can you remove these 3 lines as they are already handled in the proxy.conf?
https://github.com/linuxserver/docker-swag/blob/master/root/defaults/proxy.conf
Thanks

Comment (aptalca) [reply homeassistant.subdomain.conf.sample:41]: And include proxy.conf instead like above?

File: homeassistant.subdomain.conf.sample [hunk 2]
```diff
@@ -36,4 +36,12 @@ server {
         proxy_pass $upstream_proto://$upstream_app:$upstream_port;
 
     }
+    
+    location /api {
+        include /config/nginx/proxy.conf;
```
Comment (nemchik) [inline homeassistant.subdomain.conf.sample:41]:
```suggestion
        include /config/nginx/proxy.conf;
        resolver 127.0.0.11 valid=30s;
```

**Review by nemchik — CHANGES_REQUESTED** [review]

**Review by nemchik — APPROVED** [review]

**Pull request merged**